stringer.go

```diff
@@ -240,18 +241,57 @@ func (pkg *Package) check(fs *token.FileSet, astFiles []*ast.File) {
 }
 
 func (g *Generator) transformValueNames(values []Value, transformMethod string) {
-	var sep rune
+	var fn func(src string) string
 	switch transformMethod {
 	case "snake":
-		sep = '_'
+		fn = func(s string) string {
+			return strings.ToLower(name.Delimit(s, '_'))
+		}
+	case "snake_upper", "snake-upper":
```

Comment:
I would rather have only one way to specify the transformation option. This way the public interface is less confusing.
Could you please only accept the dashed version?
Here is the suggestion:
```suggestion
	case "snake-upper":
```